source/reviews/standard.txt

```diff
@@ -0,0 +1,90 @@
+======================
+The Standard of Review
```

Comment:
Contents of this section seem more like "Review Process" rather than a review since it involves both the reviewer and reviewee's tasks. Maybe "Standards for the Review Process" could work.